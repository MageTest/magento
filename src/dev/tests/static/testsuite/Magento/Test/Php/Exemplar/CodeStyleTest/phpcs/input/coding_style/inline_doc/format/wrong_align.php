<?php
/**
 * Short description...
  *
 * License notice...
 *
 *  @package     Magento_Something
 * @copyright   ...
 * @license     ...
 */
class Magento_Test_Php_Exemplar_CodeStyleTest_phpcs_input_coding_style_inline_doc_format_wrong_align
{
    /**
     * Function that does something
      *
     *@param string $someParam
     * @return string
     */
    public function doSomething($someParam)
    {
        return 'Hello, ' . $someParam;
    }
}
