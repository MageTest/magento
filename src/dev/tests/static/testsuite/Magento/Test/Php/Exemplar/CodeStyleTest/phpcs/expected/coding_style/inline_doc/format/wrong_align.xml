<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="4"/>
    <error line="8"/>
    <error line="16"/>
    <error line="17"/>
</config>
